src/components/views/dialogs/DevtoolsDialog.tsx

```diff
@@ -420,7 +472,7 @@ class RoomStateExplorer extends React.PureComponent {
                         { JSON.stringify(this.state.event.event, null, 2) }
                     </SyntaxHighlight>
                 </div>
-                <div className="mx_Dialog_buttons">
+                <div className="mx_Dialogbuttons">
```

Comment:
accidental class change?

Reply:
Very strange, somehow renaming the `_buttons()` method seems to have hit these as well... Normally I would use the IDE method rename, so a bit surprised it hit these classes... 🤔